guides/request-validation-csharp/example-3/example-3.4.x.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Configuration;
+using System.Web;
+using System.Net;
+using System.Web.Mvc;
+using Twilio;
+using Twilio.TwiML;
+
+namespace helloworldcsharp
+{
+	public class RequestValidatorFilterAttribute : ActionFilterAttribute
+	{
+
+		private static readonly Lazy<RequestValidator> TwilioRequestValidator
+			= new Lazy<RequestValidator>(() => new RequestValidator());
+
+		private readonly bool IsTestEnvironment;
+
+		public RequestValidatorFilterAttribute ()
+		{
+			this.IsTestEnvironment = bool.Parse(ConfigurationManager.AppSettings["IsTestEnvironment"]);
```

Comment:
I think we can use the approach proposed before to avoid coupling this implementation with the name `IsTestEnvironment `. What do you think?